Cargo.toml

```diff
@@ -14,13 +14,13 @@ poise = "0.6.1"
 tracing = "0.1.40"
 tracing-subscriber = "0.3.18"
 tokio = { version = "1.41.0", features = ["full"] }
-# uuid = { version = "1.8.0", features = ["v4"] }
+uuid = { version = "1.11.0", features = ["v4"] }
 
 [dependencies.serenity]
 version = "0.12.2"
 # features = ["voice"]
 
-# [dependencies.sqlx]
-# version = "0.7.4"
-# default-features = false
-# features = ["runtime-tokio", "tls-rustls", "macros", "postgres", "migrate"]
```

Comment:
Don't need TLS because we're only using local databases for development and on the server.